cvat-ui/src/components/create-task-page/advanced-configuration-form.tsx

```diff
@@ -416,6 +418,32 @@ class AdvancedConfigurationForm extends React.PureComponent<Props> {
         );
     }
 
+    private renderUploadChunkSize(): JSX.Element {
+        return (
+            <CVATTooltip
+                title={(
+                    <>
+                        Defines max file size sent with one request in megabytes.
+                        <br />
+                        Minimum value: 1mb
+                        <br />
+                        Maximum value: 1000mb
+                        <br />
+                        Recommended value: 100mb
+                    </>
+                )}
+            >
+                <Form.Item
+                    label='Upload chunk size'
+                    name='uploadChunkSize'
+                    rules={[{ validator: isInteger({ min: 1, max: 1000 }) }]}
+                >
+                    <Input size='large' type='number' />
+                </Form.Item>
+            </CVATTooltip>
+        );
+    }
```

Comment:
I do not think we should have this setting for a user. We need it only for tests. 
For a user it is totally useless.

Just add a setting to cvat-core: ``cvat-core/src/config.js`` and expose CVAT core library to global space ``(window || global).cvat = cvat;`` here ``cvat-ui/src/cvat-core-wrapper.ts``. Then you can access it from cypress code.

Reply:
Done.